docs/user/polarization.rst

```diff
@@ -348,39 +348,63 @@ streams.
+  As in the case of PV `PolarizerFlipper`, the analyzer flipper may change state
+  and thus the sample logs is a time-series.
+
+The following tables summarize the possible PVs values and the corresponding Sz states.
+
++-----------------------+---------------------+-------------+
+| PV Polarizer          | PV PolarizerFlipper |  Sz State   |
++=======================+=====================+=============+
+| 0 - No Polarizer      |                     | unpolarized |
++-----------------------+---------------------+-------------+
+| 1 - Reflection        | 1 - ON              | up          |
++-----------------------+---------------------+-------------+
+| 1 - Reflection        | 0 - OFF             | down        |
++-----------------------+---------------------+-------------+
+| 2 - Transmission      | 1 - ON              | down        |
++-----------------------+---------------------+-------------+
+| 2 - Transmission      | 0 - OFF             | up          |
++-----------------------+---------------------+-------------+
+| 3 - Undefined         |                     | unpolarized |
++-----------------------+---------------------+-------------+
+
++-----------------------+--------------------+--------------+
+| PV Analyzer           | PV AnalyzerFlipper | Sz State     |
++=======================+====================+==============+
+| 0 - No Analyzer       |                    | unpolarized  |
++-----------------------+--------------------+--------------+
+| 1 - Fan & SF2         | 1 - ON             | up           |
++-----------------------+--------------------+--------------+
+| 1 - Fan & SF2         | 0 - OFF            | down         |
++-----------------------+--------------------+--------------+
+| 2 - 3He               | 1 - ON             | down         |
++-----------------------+--------------------+--------------+
+| 2 - 3He               | 0 - OFF            | up           |
++-----------------------+--------------------+--------------+
+| 3 - Undefined         |                    | unpolarized  |
++-----------------------+--------------------+--------------+
+
+There is a adjustment time period when a flipper changes value during which the state of the flipper is uncertain.
+Events collected during this time period cannot be assigned to either of the "ON" or "OFF" states,
+hence they should be discarded in the reduction.
+The set of time periods for the polarizer flipper are collected in PV `PolarizerVeto`
+and for the analyzer flipper in PV `AnalyzerVeto`.
+
+The image below shows the relation between the flipper and veto logs. Whenever the flipper changes state,
+there's a time period during which the veto log becomes 1.
+During reduction, we'll use Mantid's filtering capabilities to discount events collected during these time periods.
+
+.. figure:: media/polarization_2.png
+   :alt: Relation between flipper and veto logs
+   :width: 400px
```

Comment:
the image could be a bit wider to fill more of the width of the column

Reply:
I'll double it's width